### DISCUSSION TITLE
Image: macOS-14, Xcode 16.0 not found

### DISCUSSION THREAD

**Discussion by surendrapatel-rak** [actions/virtual-environments#10928] [Installed software]
Some Azure pipeline agent we are giving following error. Any idea ?

On: Image: macOS-14

/usr/bin/xcodebuild -version
xcrun: error: missing DEVELOPER_DIR path: /Applications/Xcode_16.app

**Comment by Cheesebaron** [comment]
See: https://github.com/actions/runner-images/issues/10703

**Comment by v-aartis17** [comment]
It's expected behaviour. Consider to switch to macOS-15 images, please, in order to use Xcode 16.
please see [announcement](https://github.com/actions/runner-images/issues/10703)
